baseapp/server.go

```diff
@@ -76,7 +77,11 @@ func NewServer(c HTTPConfig, params ...Param) (*Server, error) {
 	}
 
 	if base.server == nil {
-		base.server = &http.Server{}
+		base.server = &http.Server{
+			TLSConfig: &tls.Config{
+				MinVersion: tls.VersionTLS12,
+			},
+		}
```

Comment:
Doesn't this mean that if you don't specify a server you'll always get TLS, even when testing locally?

Reply:
no, it should only mean that when you have TLS configured in YAML, it will have MinVersion set for you

Reply:
Ok. Merge away